internal/provisioner/plain/main.go

```diff
@@ -74,7 +77,8 @@ func main() {
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.BoolVar(&rukpakVersion, "version", false, "Displays rukpak version information")
-	flag.StringVar(&gitClientImage, "git-client-image", "alpine/git:v2.32.0", "Configures which git container image to use to clone bundle git repos")
+	flag.StringVar(&storageDirectory, "storage-dir", storage.DefaultBundleCacheDir, "Configures the directory that is used to store Bundle contents.")
```

Comment:
Even though we set the default here correctly, do you think it's worthwhile setting `--storage-dir` explicitly in the deployment spec so that if we ever decide to change the volumeMount, its more obvious that we also need to change this flag's value?

Reply:
Yeah I had already made this change but it looks like it got lost while cherrypicking this change to it's own branch. I'll force push those deployment manifest changes in a second.